### ISSUE TITLE
Stale docs: Logging uncaught exceptions

### ISSUE THREAD

**Issue by erights** [endojs/endo#887]
At https://github.com/endojs/endo/blob/master/packages/ses/src/error/README.md

>  * SES hides annotations and stack traces by default. To reveal them, use a mechanism like `process.on("uncaughtException")` in Node.js to catch the error and log it back to the `console` tamed by `lockdown`.

@kriskowal could you update this in light of your recent tooling improvements? Thanks.

**Comment by erights** [comment]
@kriskowal still relevant, or can it be closed?

**Comment by kriskowal** [comment]
Yes, this is done. I can’t say we’ve uttered the last word about this, but the basic need is satisfied. The mechanism would ideally be less coupled or the platform binding more readily swapped in or out, but that is another issue.
